mungegithub/mungers/cherrypick-label-unapproved.go

```diff
@@ -29,12 +29,13 @@ import (
 )
 
 const (
-	labelUnapprovedPicksName = "label-unapproved-picks"
-	labelUnapprovedFormat    = "This PR is not for the master branch but does not have the `%s` label. Adding the `%s` label."
+	cherrypickUnapprovedLabel = doNotMergePrefix + "/cherry-pick-not-approved"
```

Comment:
I believe we should migrate these labels from the old to new format:

a) Merge a version of mungegithub that understands both `foo` and `do-not-merge/foo` variants of each label
b) Deploy this version
c) Migrate all the labels across the org
d) Merge a version of mungegithub that removes the logic for deprecated `foo`
e) Deploy this version

Reply:
The way I saw it on these was:

 - currently the mungers add `do-not-merge`
 - this will remain blocking with this patch and therefore we are backwards compatible
 - none of the mungers ever remove one of these labels
 - therefore, there is diff between a) and d)

Reply:
Are you sure all the holds currently add the `do-not-merge` label? If so SGTM